testing-overhead/src/test/java/io/opentelemetry/OverheadTests.java

```diff
@@ -71,4 +84,10 @@ void runAppOnce(TestConfig config, Agent agent) throws Exception {
 
     //TODO: Parse and aggregate the test results.
   }
+
+  private void writeStartupTimeFile(Agent agent, long start) throws IOException {
```

Comment:
Startup time of Spring applications can be parsed from the logs. Current approach of relying on testcontainers check depends on the frequency of polling, so has higher granularity

Reply:
True. I'm not currently pulling in the logs, so the time isn't readily available for collection. I could look into that, but I honestly don't think we should be too concerned with subsecond granularity. We could also possibly make testcontainers poll more than 1s. 

If you think it's important can you please open an issue to track it?  Thanks!